pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -113,20 +113,29 @@ func InitCnsOperator(configInfo *types.ConfigInfo, coInitParams *interface{}) er
 		return err
 	}
 
-	if cnsOperator.coCommonInterface.IsFSSEnabled(ctx, common.FileVolume) {
-		// Create CnsFileAccessConfig CRD from manifest if file volume feature is enabled
-		err = k8s.CreateCustomResourceDefinitionFromManifest(ctx, "cnsfileaccessconfig_crd.yaml")
-		if err != nil {
-			log.Errorf("Failed to create %q CRD. Err: %+v", apis.CnsFileAccessConfigPlural, err)
-			return err
-		}
-		// Create FileVolumeClients CRD from manifest if file volume feature is enabled
-		err = k8s.CreateCustomResourceDefinitionFromManifest(ctx, "cnsfilevolumeclient_crd.yaml")
-		if err != nil {
-			log.Errorf("Failed to create %q CRD. Err: %+v", internal.CnsFileVolumeClientPlural, err)
-			return err
-		}
+	// TODO:
+	// Explore how CRD creation can be protected using feature state flags
+	// Right now, the init() method in pkg/syncer/cnsoperator/controller/add_cnsfileaccessconfig.go gets invoked always during CnsOperator Init
```

Comment:
I looked at that file. Can we not check for the FSS there before appending to `AddToManagerFuncs`? Its not a good idea to create the CRD all the time(without FSS)

Reply:
>  Its not a good idea to create the CRD all the time(without FSS)
I agree, but as a short term solution to unblock test automation on Supervisor and TKG clusters I was thinking of pushing this code change.

As mentioned in `TODO` I would like to explore on how to add FSS for CRD creation and work on it as a separate item. I believe this will add changes to other controllers we have in the repo and would require some extra effort.

If that is an agreeable reasoning, see if this can be pushed for now.
cc: @SandeepPissay

Reply:
I tried incorporating the FSS check in the init of cnsfileaccessconfig controller. This should address the review comment.
PTAL